[IMP] Update l10n_ph Documentation for 18.0 (#11531)
odoo/documentation: content/applications/finance/fiscal_localizations/philippines.rst

BIR 2307 report
===============

> **BIR 2307** report data, also known as *Certificate of Creditable Tax Withheld at Source*,

Author: Hi @dade-odoo Yes I removed it since the old link was broken. I was thinking to avoid future broken links by reducing number of externally linked documents and also we are currently working on generating the PDF directly in Odoo instead of user to manually fill it in. I think just future proofing a bit as the link may not be needed in the future!

> **BIR 2307** report data, also known as *Certificate of Creditable Tax Withheld at Source*,

Reviewer: I like the idea of removing the link when it's no longer needed, but it seems like at this point, it is still necessary. Once we have the ability to generate the PDF directly in Odoo we'll probably be updating this page again anyway, so that would be a good time to remove the link if you're worried about it breaking again in the future. For now, can you replace it with a working link? Thanks!

> **BIR 2307** report data, also known as *Certificate of Creditable Tax Withheld at Source*,

Author: Thanks! I have added the new link back into the file! I also squashed and commit to this PR hopefully it was successful! Let me know if there is further changes needed!

.. important::
   Odoo cannot generate the 2550Q BIR formatted PDF report directly. It should be used as a
   reference when externally filing the form manually or online.

QAP & SAWT Report
=================

The **QAP**, also known as the *Quarterly Alphalist of Payees* report and the **SAWT**, also known
as the *Summary Alphalist of Withholding Tax* report can be viewed and exported (in XLSX format).
The report can be viewed from :menuselection:`Reporting --> Tax Return`. In the Tax Return,
switch the report to select :menuselection:`Report: --> SAWT & QAP (PH)`.

The report is split into two sections, which can be accessed from their respective buttons at the
top:

> - | :guilabel:`SAWT` for :abbr:`SAWT (Summary Alphalist of Withholding Tax)` report

Reviewer: ```suggestion
- | :guilabel:`SAWT` for the :abbr:`SAWT (Summary Alphalist of Withholding Tax)` report
```

  | All customer invoices with the associated sales witholding taxes applied are
   shown in this report.

> - | :guilabel:`QAP` for :abbr:`QAP (Quarterly Alphalist of Payees)` report

Reviewer: ```suggestion
- | :guilabel:`QAP` for the :abbr:`QAP (Quarterly Alphalist of Payees)` report
```

  | All vendor bills with the associated purchase witholding taxes applied are shown in this report.

.. image:: philippines/philippines-sawt.png
   :alt: SAWT & QAP Report

.. important::
   Odoo cannot generate the DAT files directly. The :guilabel:`Export SAWT & QAP` and
   :guilabel:`XLSX` buttons export an XLSX file, which can be processed using an *external* tool to
   convert to the DAT format.

Check Printing
==============

Philippines check print layout is following the latest

> :abbr:`PCHC (Philippine Clearing House Corporation)` standardised printing format.

Reviewer: ```suggestion
:abbr:`PCHC (Philippine Clearing House Corporation)` standardized printing format.
```
We use the American English spelling with "z" instead of "s" :)

> For enabling check printing, under :menuselection:`Accounting --> Configuration --> Settings`,
> ensure that the :guilabel:`Checks` setting is enabled and to choose the correct
> :guilabel:`Check Layout` as `Print Check - PH`.

Reviewer: ```suggestion
To enable check printing, under :menuselection:`Accounting --> Configuration --> Settings`, ensure
that the :guilabel:`Checks` setting is enabled and to choose the correct :guilabel:`Check Layout` as
`Print Check - PH`.
```

.. image:: philippines/philippines-check-setup.png

>    :alt: Print Check - PH settings

Reviewer: No need to include this image :)

Reviewer: Ah I didn't see that you had 5 commits here. So yes, you can just remove these three lines and the push the commit again, but you'll have to squash them into just one single commit for the PR.

> The workflow to print the check is following the standard workflow which can be followed here:
> :doc:`Pay by Checks <../accounting/payments/pay_checks>`.

Reviewer: ```suggestion
Print checks using the :ref:`standard workflow <../accounting/payments/pay_checks>`.
```
